content/gsoc/2024/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: PyBaMM GSoC 2024 Project Ideas
+draft: true
```

Comment:
```suggestion
draft: false
```

We can change this to false whenever we are ready to showcase this page for potential participants.

Reply:
Removed this since we are keeping just the page for now

Reply:
Reverted this since we are keeping just the page for now